code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_powers.dm

```diff
@@ -10,16 +10,15 @@
 	if (!check_and_use_plasma_owner())
 		return
 
-	animate(xeno, alpha = alpha_amount, time = 0.1 SECONDS, easing = QUAD_EASING)
-	button.icon_state = "template_active"
-	xeno.update_icons() // callback to make the icon_state indicate invisibility is in lurker/update_icon
+	animate(Xeno, alpha = alpha_amount, time = 0.1 SECONDS, easing = QUAD_EASING)
+	Xeno.update_icons() // callback to make the icon_state indicate invisibility is in lurker/update_icon
 
-	xeno.speed_modifier -= speed_buff
-	xeno.recalculate_speed()
+	Xeno.speed_modifier -= speed_buff
+	Xeno.recalculate_speed()
 
-	if (xeno.mutation_type == LURKER_NORMAL)
-		var/datum/behavior_delegate/lurker_base/behavior = xeno.behavior_delegate
-		behavior.on_invisibility()
+	if (Xeno.mutation_type == LURKER_NORMAL)
+		var/datum/behavior_delegate/lurker_base/BD = Xeno.behavior_delegate
+		BD.on_invisibility()
```

Comment:
why

Reply:
sorry mb i did this PR before that was changed and forgor to update that part